openfecwebapp/api_caller.py

```diff
@@ -48,3 +49,6 @@ def load_cmte_financials(committee_id):
 def install_cache():
     import requests_cache
     requests_cache.install_cache()
+
+def limit_by_amount(curr_url, amount):
+    return curr_url + '?page=1&per_page=' + str(amount * 2)
```

Comment:
Prefer URL libraries over string concatenation for building URLs, e.g.
```python
from urllib.parse import urlencode

def limit_by_amount(curr_url, amount):
    query = urlencode({'page': 1, 'per_page': amount * 2})
    return '{0}?{1}'.format(curr_url, query)
```

Which takes care of type-casting and joining for you.